Source/WTF/wtf/StdLibExtras.h

```diff
@@ -1091,6 +1091,13 @@ ALWAYS_INLINE constexpr void forEachSetBit(std::span<const WordType> bits, size_
     }
 }
 
+template<typename T, typename U>
+ALWAYS_INLINE void initializeOnce(const std::unique_ptr<T>& ptr, std::unique_ptr<U>&& obj)
```

Comment:
We have a bit of a name conflict with call_once and dispatch_once. In those functions, "once" promises mutual exclusion. 

Maybe we should just call this "initialize"? "Initialize" describes a starting condition, which already helps to convey that it's an error to initialize twice.

Reply:
Maybe we can call it `lazilyInitialize`? `initialize` is too generic of a word and it prevents easy grep'ing.

Reply:
I like lazilyInitialize or lazyInitialize, given what it's meant to be used for.

Reply:
I'd say lazyInitialize.